tools/CMakeLists.txt

```diff
@@ -3,3 +3,4 @@
 #
 
 add_subdirectory(revng-lift)
+add_subdirectory(revng-autoenforcer)
```

Comment:
Why do we still have autoenforcer here?

Reply:
it was left behind when we were deciding how to call it properly, the name pipeline was not really great, we can prefix it with revng-pipeline but then the target becomes revng-revng-pipeline

Reply:
wait, no, the target would still be revng-pipeline. 

if that is good enough i will keep it